client/src/components/settings/theme.tsx

```diff
@@ -1,13 +1,18 @@
 import { Form } from '@freecodecamp/react-bootstrap';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import store from 'store';
+import { playTone } from '../../utils/tone';
 
 import ToggleSetting from './toggle-setting';
 
+export enum Themes {
+  Night,
+  Default
+}
+
```

Comment:
So this seems to break things.

The `enum` stores this as `Themes.Night` as `0` and `Themes.Default` as `1`. I'm not sure why, but this leaves my client permanently in night mode - presumably because the database schema doesn't match this?

Reply:
Oops. That is my bad, because I changed it too many times 😅